lang_python.adoc

```diff
@@ -33,68 +33,94 @@ Runtime library API version must match compiler version that you're using
 
 [[add-runtime-release]]
-==== Using release version
+==== Using release version (NOT recommended)
 
 With stable version of Kaitai Struct compiler, one can use use following
 methods:
 
 * On Debian / Ubuntu, one can install it from apt repositories:
 
-** For Python 2.x:
+** For Python 3.x:
 +
 [source,shell]
-sudo apt-get install python-kaitaistruct
+sudo apt-get install python3-kaitaistruct
 +
-Reference: https://packages.debian.org/buster/python-kaitaistruct[package info for Debian], https://packages.ubuntu.com/eoan/python-kaitaistruct[package info for Ubuntu]
+Reference: https://packages.debian.org/buster/python3-kaitaistruct[package info for Debian], https://packages.ubuntu.com/eoan/python3-kaitaistruct[package info for Ubuntu]
 
-** For Python 3.x:
+** For Python 2.x (DEPRECATED, python 2 is EOL since 2020-01-01):
 +
 [source,shell]
-sudo apt-get install python3-kaitaistruct
+sudo apt-get install python-kaitaistruct
 +
-Reference: https://packages.debian.org/buster/python3-kaitaistruct[package info for Debian], https://packages.ubuntu.com/eoan/python3-kaitaistruct[package info for Ubuntu]
+Reference: https://packages.debian.org/buster/python-kaitaistruct[package info for Debian], https://packages.ubuntu.com/eoan/python-kaitaistruct[package info for Ubuntu]
+
 
-* If your project is using *requirements.txt* to manage your
-  dependencies, add the following line to it:
+* Add the following dependency specifier into
+`install_requires` of `options` section
+of your `setup.cfg` (or the arg of `setuptools.setup`
+with the same name in your `setup.py`) :
 +
 ----
 kaitaistruct
 ----
 +
-and then run `pip install -r requirements.txt` to update all your
-dependencies.
+and then run `pip install --upgrade -e ./` in the dir with the `setup.py`
+in order to install your package and all its dependencies in
+an editable way.
 
-* Otherwise, you can just install the package manually using *pip*
+* Otherwise, you can just install the package manually using `pip`
   (Python package manager):
 +
 [source,shell]
-pip install kaitaistruct
+pip install --upgrade kaitaistruct
 
 [[add-runtime-snapshot]]
-==== Using latest (snapshot) version
+==== Using latest (snapshot) version (RECOMMENDED)
 
 If you're using latest (unreleased AKA "unstable" AKA "snapshot") build
 of Kaitai Struct compiler, that will require latest runtime libraries as
 well:
 
-* If your project is using *requirements.txt* to manage your
-  dependencies, add the following line to it:
+* You can a make `pip` to use a bleeding edge version of the runtime by
+  adding a https://www.python.org/dev/peps/pep-0508/[PEP 508]
+  dependency specifier into `install_requires`:
 +
 ----
-kaitaistruct @ git+https://github.com/kaitai-io/kaitai_struct_python_runtime.git
+    kaitaistruct @ git+https://github.com/kaitai-io/kaitai_struct_python_runtime.git
 ----
 +
-and then run `pip install -r requirements.txt` to update all your
-dependencies.
+.
```

Comment:
I think this line can be removed? Or does this have some special meaning in asciidoc that I'm not aware of?

Reply:
It's just here to add a full stop.